src/pubkey.h

```diff
@@ -291,6 +291,30 @@ class XOnlyPubKey
     SERIALIZE_METHODS(XOnlyPubKey, obj) { READWRITE(obj.m_keydata); }
 };
 
+/** An ElligatorSwift-encoded public key. */
+struct EllSwiftPubKey
+{
+public:
+    static constexpr size_t ELLSWIFT_ENCODED_SIZE = 64;
```

Comment:
tiny nit: could be put in the "private:" section, since there is a static constexpr `size()` function returning this value below anyway.